test/colony-network-mining.js

```diff
@@ -793,7 +793,7 @@ contract("ColonyNetworkMining", accounts => {
       );
     });
 
-    it('should not allow "startNextCycle" to be called if a cycle is in progress', async () => {
+    it('should not allow "repCycle.nSubmittedHashesCycle" to be called if a cycle is in progress', async () => {
```

Comment:
Is this a mass rename gone wrong? The original test title was correct.

Reply:
I have absolutely no clue what happened here 😕